rai/client_test.go

```diff
@@ -60,6 +62,65 @@ func findModel(models []Model, name string) *Model {
 	return nil
 }
 
+func getConfig(cfg *Config) error {
+	configPath, _ := expandUser(DefaultConfigFile)
+	if _, err := os.Stat(configPath); err == nil {
+		return LoadConfig(cfg)
+	} else {
+		clientId := os.Getenv("CLIENT_ID")
+		clientSecret := os.Getenv("CLIENT_SECRET")
+		clientCredentialsUrl := os.Getenv("CLIENT_CREDENTIALS_URL")
+		raiHost := os.Getenv("HOST")
+		if raiHost == "" {
+			raiHost = "azure.relationalai.com"
+		}
+
+		configFormat := `
+		[default]
+		host=%s
+		region=us-east
+		port=443
+		scheme=https
+		client_id=%s
+		client_secret=%s
+		client_credentials_url=%s
+		`
+		configSrc := fmt.Sprintf(configFormat, raiHost, clientId, clientSecret, clientCredentialsUrl)
+		return LoadConfigString(configSrc, "default", cfg)
+	}
+}
```

Comment:
we have similar behavior here https://github.com/RelationalAI/rai-sdk-go/blob/main/rai/main_test.go#L113-L139
if we can refactor that code out and re-use it in `TestNewClient` and `newTestClient` that would be awesome

Reply:
Resolved. Moved it to main_test.go and just use it in both places. Moving forward may consider having a separate file to put all the shared functions.